mysql-test/suite/galera/t/MDEV-16509.test

```diff
@@ -15,14 +15,13 @@ CREATE TABLE t1 (f1 INT PRIMARY KEY) ENGINE=InnoDB;
 # Scenario 1: Block INSERT after commit order release after queued for
 # group commit. Verify that
 #
-# - WSREP_LAST_SEEN_GTID is not advanced before commit finishes
```

Comment:
What is the use case for `WSREP_LAST_SEEN_GTID` that isn't served by existing [gtid variables](https://mariadb.com/kb/en/library/gtid/#system-variables)?

Reply:
Explanation from first conversation can be used. WSREP_LAST_SEEN_GTID should show latest GTID in cluster.